extensions/amp-sidebar/0.1/test/integration/test-amp-sidebar.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {listenOncePromise} from '../../../../../src/event-helper';
+
+const config = describe.configure().ifNewChrome();
+
+config.run('amp-sidebar', function() {
+  this.timeout(100000);
```

Comment:
Is `this.timeout` necessary to prevent flakes, or was it duplicated from somewhere else? If we can do without it, I think we should